platform.md

```diff
@@ -754,7 +754,7 @@ If the launcher errors before executing the process it will have one of the foll
 |-----------|-------|
 | `11`      | Platform API incompatibility error
 | `12`      | Buildpack API incompatibility error
-| `700-799`|  Launch-specific lifecycle errors
+| `200-219`|  Launch-specific lifecycle errors
```

Comment:
if launch is a different process, does it have to abide by the same incrementing number here?

Reply:
This is a good question. I imagine you could say the same for rebaser. I left it as consecutive ranges because that's how it was before, but happy to change if that's what we decide.